bin/getInfoRippled.sh

```diff
@@ -0,0 +1,47 @@
+ #!/bin/bash
+
+export TMP_LOC="/tmp/ripple_info"
```

Comment:
Consider `TMP_LOC=$(mktemp -d ripple_info.XXXX)`. This will create a temporary directory for you and return the directory name.